app/Models/Individual.php

```diff
@@ -341,7 +348,7 @@ public function upcomingContractedProjects(): MergedRelation
      *
      * @return bool
      */
-    public function isPublishable(): bool
+    public function isPublishable(bool $ignoreSuspension = false): bool
```

Comment:
I'm guessing this parameter isn't necessary. Similar for other cases it is added.

Reply:
Yes, I removed them.